gensim/test/test_word2vec.py

```diff
@@ -275,6 +275,13 @@ def test_persistence(self):
         self.assertTrue(np.allclose(wv.vectors, loaded_wv.vectors))
         self.assertEqual(len(wv), len(loaded_wv))
 
+    def test_persistence_backwards_compatible(self):
+        """Can we still load a model created with an older gensim version?"""
+        path = datapath('model-from-gensim-3.8.0.w2v')
+        model = word2vec.Word2Vec.load(path)
+        x = model.score(['test'])
+        assert x is not None
```

Comment:
Should probably be a `self.assertSomething()` call, rather than a language `assert` for both consistency with other tests & rare case where someone may have disabled language-`assert`s via command-line. And, relying on the (maybe not possible to support indefinitely) `score()` function to probe model completeness, when that's not declared as tested in method name, could create maintenance headaches at some later date.